### PR TITLE
Function fix

### PR DESCRIPTION
**Pull request by ldm0**
Author maybe a little sleepy when inplementing the mvwaddch function : )

### CODE REVIEW

File: wincurses.c [hunk 1]
```diff
@@ -284,8 +284,9 @@ int mvaddch(int y, int x, const chtype ch)
 
 int mvwaddch(WINDOW *win, int y, int x, const chtype ch)
 {
-	move(y, x);
-	return waddch(stdscr, ch);
+	if(wmove(win,y, x)==ERR)
```
Comment (justinmeza) [inline wincurses.c]: Looks good, the only change I would make is formatting:
```
if (wmove(win, y, x) == ERR)
```

**Review by justinmeza — CHANGES_REQUESTED** [review]
Thanks for updating this! I just have one minor request regarding formatting.

File: wincurses.c (hunk 1 above)
Comment (ldm0) [reply wincurses.c]: Format has been corrected! : ) This is my first pull request and I hope that I am doing the right thing.

**Review by justinmeza — APPROVED** [review]
Thanks!

**Pull request merged**